.github/ISSUE_TEMPLATE/proposal--featured-talk.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Talk proposal - Feature Talk (max 25 minutes)
+about: Propose a feature talk. Max 25 minutes in length
+title: "Talk proposal: <talk title>"
+labels: Feature
+assignees: ""
+---
+
+# Title of talk
+
+A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine. Please avoid the content like product pitches or vendor driven talk. 
```

Comment:
```suggestion
A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine. Please avoid content like product pitches or vendor-driven talks. 
```